### ISSUE TITLE
Add `jax.random` functions to Jax frontend

### ISSUE THREAD

**Issue by hirwa-nshuti** [unifyai/ivy#8513]
Adding the [jax.random package](https://jax.readthedocs.io/en/latest/jax.random.html#list-of-available-functions) functions in the Jax frontend.

- [ ] PRNGKey
- [ ] ball
- [ ] bernoulli
- [ ] beta
- [ ] categorical
- [ ] cauchy
- [ ] choice
- [ ] drichlet
- [ ] double_sided_maxwell
- [ ] exponential
- [ ] fold_in
- [ ] gamma
- [ ] generalized_normal
- [ ] gumbel
- [ ] loggamma
- [ ] logistic
- [ ] maxwell
- [ ] multivariate_normal
- [ ] normal
- [ ] orthogonal
- [ ] pareto
- [ ] permutation
- [ ] poisson
- [ ] rademacher
- [ ] randint
- [ ] shuffle
- [ ] split
- [ ] t
- [ ] truncated_normal
- [ ] uniform
- [ ] weibull_min

**Comment by bmondal98** [comment]
- [ ] #8822

**Comment by Arrabonae** [comment]
randint #8974

**Comment by andyjaramillo** [comment]
#9406

**Comment by aymuos15** [comment]
- [ ] #9476

**Comment by PyroCandy** [comment]
#9530

**Comment by adityagandhamal** [comment]
- [ ] #9796

**Comment by sagewhocodes** [comment]
- [ ] #9799

**Comment by xingyu-long** [comment]
- [ ] #11083

**Comment by adityagandhamal** [comment]
- [ ] #12244

**Comment by eigengravy** [comment]
#12762

**Comment by MuhammedAshraf2020** [comment]
- [ ] #13396

**Comment by rodlaf** [comment]
- [ ] #13464

**Comment by rodlaf** [comment]
- [ ] #13465

**Comment by MuhammedAshraf2020** [comment]
- [ ] #13482

**Comment by MuhammedAshraf2020** [comment]
- [ ] #13483

**Comment by MuhammedAshraf2020** [comment]
- [ ] #13484

**Comment by MuhammedAshraf2020** [comment]
- [ ] #13497

**Comment by MuhammedAshraf2020** [comment]
- [ ] #13498

**Comment by DJALLIL05** [comment]
- [ ] #13657

**Comment by MuhammedAshraf2020** [comment]
- [ ]  #13919

**Comment by BrookeMa** [comment]
- [ ] #13930

**Comment by Agecaf** [comment]
- [ ] #14017

**Comment by MuhammedAshraf2020** [comment]
- [ ] #14219

**Comment by Arronfox** [comment]
shuffle #14396

**Comment by hnehme99** [comment]
- [ ] #14407

**Comment by MuhammedAshraf2020** [comment]
- [ ] #14408

**Comment by MuhammedAshraf2020** [comment]
- [ ] #14409

**Comment by MuhammedAshraf2020** [comment]
- [ ] #14478

**Comment by MuhammedAshraf2020** [comment]
- [ ] #14479

**Comment by MuhammedAshraf2020** [comment]
- [ ] #14480

**Comment by MuhammedAshraf2020** [comment]
- [ ] #14481

**Comment by MuhammedAshraf2020** [comment]
- [ ] #14482

**Comment by MuhammedAshraf2020** [comment]
- [ ] #14483

**Comment by TashfikS** [comment]
- [ ]  #14672

**Comment by iammarco11** [comment]
permutation #14840

**Comment by iammarco11** [comment]
- [ ] #14840

**Comment by iammarco11** [comment]
- [ ] #14858

**Comment by iammarco11** [comment]
- [ ] #14881

**Comment by iammarco11** [comment]
- [ ]  #14882

**Comment by NouranYoussef** [comment]
#15354

**Comment by baberabb** [comment]
- [ ] #15406

**Comment by NouranYoussef** [comment]
#15413

**Comment by NouranYoussef** [comment]
#15429

**Comment by NouranYoussef** [comment]
#15432

**Comment by NouranYoussef** [comment]
#15433

**Comment by baberabb** [comment]
- [ ] #15655

**Comment by Nordiniv** [comment]
- [ ] #15755

**Comment by Nordiniv** [comment]
I want to add ivy.choice to the ivy functional backend.

**Comment by dash96** [comment]
- [ ]  double_sided_maxwell #16129

**Comment by declanelias** [comment]
weibull_min #16353

**Comment by baberabb** [comment]
- [ ] #16545

**Comment by ryanacs** [comment]
#16697

**Comment by z1chance** [comment]
https://github.com/unifyai/ivy/issues/14881

**Comment by Zaituny** [comment]
- [ ] #17755

**Comment by stalemate1** [comment]
- [ ] #17774

**Comment by rubythalib33** [comment]
#18256

**Comment by stalemate1** [comment]
orthogonal https://github.com/unifyai/ivy/issues/17774

**Comment by stalemate1** [comment]
- [ ] double_sided_maxwell #18931

**Comment by stalemate1** [comment]
- [ ] maxwell #19347

**Comment by stalemate1** [comment]
- [ ] double_sided_maxwell #19483

**Comment by Muradmustafayev-03** [comment]
orthogonal #20273

**Comment by TemiOjo25** [comment]
- [ ] #20928

**Comment by VaishnaviMudaliar** [comment]
categorical #21785

**Comment by SD-13** [comment]
- [ ] #21793

**Comment by waqaarahmed** [comment]
- [ split] #22222

**Comment by waqaarahmed** [comment]
-  [bits] #22222

**Comment by waqaarahmed** [comment]
-[ ] split#21877

**Comment by waqaarahmed** [comment]
- [ ] split#21877

**Comment by TemiOjo25** [comment]
- [ ] #22047

**Comment by TemiOjo25** [comment]
- [ ] #22132

**Comment by khawajaJunaid** [comment]
#22781

**Comment by rsaeta** [comment]
- [ ] #23259

**Comment by dash96** [comment]
- [ ] #23337

**Comment by Lesedi-N** [comment]
-[ ]  #26274
